CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v1.1.3 - 2022-05-31
+
+# Changes
+- jinja2 version dependency specification modified such that versions in the 3.x release are supported.
+
```

Comment:
Looks like we missed entries for 1.1.1 and 1.1.2 - would it make sense to backfill those now?